courses/models_test.py

```diff
@@ -746,3 +748,15 @@ def test_instructors(has_page):
     assert course_run.instructors == (
         [{"name": name} for name in faculty_names] if has_page else []
     )
+
+
+def test_partner_name_is_unique():
+    """
+    Tests that case-insensitive partner name is unique.
+    """
+    PartnerFactory.create(name="Emeritus")
+
+    with pytest.raises(ValidationError):
+        PartnerFactory.create(name="emeritus")
+
+    PartnerFactory.create(name="Emirates")
```

Comment:
nit: This line/creation might not be needed. I think the above assertion is enough.